### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
https://github.com/zenml-io/zenml-dashboard/blob/feature/update-readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,177 @@
...
+If you would like to contribute, please review our [Contributing
+Guide](CONTRIBUTING.md) for all relevant details.
+
+# 🆘 Getting Help
+
+Need a helping hand? We've got you covered! Getting assistance with ZenML is quick and easy.
+
+1. **Join our Slack Community:**
+
+   - Our lively Slack community is buzzing with friendly faces and helpful discussions. Drop by, ask questions, and connect with fellow enthusiasts. [Get your invite](https://zenml.io/slack-invite/).
```
Comment (htahir1) [inline README.md:163]:
```suggestion
   - Our lively Slack community is buzzing with friendly faces and helpful discussions. Drop by, ask questions, and connect with fellow enthusiasts. [Get your invite](https://zenml.io/slack/).
```

**Comment by Cahllagerfeld** [comment]
As the readme isn't a build file and not included in the release, we can adjust this anytime we want. I'd merge it for now, to also bring the slight adjustment with the regular expression in, if you're not opposed to it

**Review by AlexejPenner — APPROVED** [review]
Approval for the changes to the regex pattern :seal:

**Review by htahir1 — APPROVED** [review]

**Comment by htahir1** [comment]
ok lets merge it @Cahllagerfeld

**Pull request merged**